### DISCUSSION TITLE
[Project Beta] Adding issues to a filtered table doesn't give you a chance to add the correct column value.

### DISCUSSION THREAD

**Discussion by aanthony1243** [community/community#7112] [Projects and Issues]
If I have a filtered table, with a custom column, when I add the issue I get a popup that says that the issue I added was done, but I can't see it because of filters.  Would be nice if we somehow got a chance to add the correct column value.  

Workaround:  if I have an "all issues" table that is grouped by a column, when I add issues there, the column is correctly populated depending on which group I add it to.  It would be useful if the filtered table had a similar pattern, i.e.  "this table has all issues where column "X == Foo"  so when I add an issue to this table, it should assign column X to "Foo"

**Comment by willhennessy** [comment]
+100 I feel this pain every day. please fix!  The issue should automatically inherit the conditions required to appear in this list.

In my use cases, those conditions typically include Status (Todo, In progress, Done) and Project Name (a custom single select field)
